Sources/Build/BuildOperation.swift

```diff
@@ -163,6 +168,7 @@ public final class BuildOperation: PackageStructureDelegate {
             ) {
                 return result
             }
+            diags.emit(error: "computing llbuild target name failed")
```

Comment:
Should probably wrap this in `!diags.hasErrors` as well? Also, maybe reword to "internal error: failed to compute the llbuild target name"